### PR TITLE
species QC to ChlRei

### PR DESCRIPTION
**Pull request by izabelcavassim**
QC for Chlamydomonas reinhardtii #890, #863, #950
- [x] fill out the test stubs
- [X] delete the pytest.mark.skip lines that make the tests not run
- [x] make sure they pass, talking to the original author to figure out discrepancies

@aays Can you explain where you got your generation time from? For now I don't know the source. 
Thanks.

### CODE REVIEW

File: tests/test_ChlRei.py [hunk 1]
```diff
@@ -34,56 +34,56 @@ class TestGenomeData(test_species.GenomeTestBase):
...
+            "13": 7.56e-10,
+            "14": 8.96e-10,
+            "15": 6.91e-10,
+            "16": 9.59e-10,
+            "17": 1.05e-09,
         }.items(),
     )
     def test_mutation_rate(self, name, rate):
+        print("This is the mutation rate")
+        print(self.genome.get_chromosome(name).mutation_rate)
```
[the remark this answers is not in the document]
Comment (izabelcavassim) [reply tests/test_ChlRei.py:88]: whats is the change here? spacing?

**Comment by izabelcavassim** [comment]
OK @aays , I am almost there, I just need to find out where you got the generation time from. Once you do give me the citation I will finish my modifications.

File: tests/test_ChlRei.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (petrelharp) [reply tests/test_ChlRei.py:88]: Ah, sorry - I was suggesting you delete the `print( )`s, since they looked like they were just left over from debugging?

**Comment by aays** [comment]
Very sorry for not getting to this sooner, the past two days were a bit more hectic than I'd have liked! Thanks very much for your comments and review so far @izabelcavassim and @petrelharp. 

The generation time was calculated from Vitova et al's estimate of the doubling time at 20 degrees Celsius of 0.10 mitoses / h (p. 604 in the paper, listed as 'doubling / h'). 

From that, I obtained 876 generations per year (0.10 doublings/h * 24 h * 365 days) hence 1/876 as the generation time. C. reinhardtii can also undergo meiosis, but is expected to have a meiotic cycle once every ~840 generations (e.g. only once a year) based on [this work of ours](https://doi.org/10.1093/gbe/evaa057).

This is the best data I know of on C. reinhardtii's generation time, I'm afraid. Please let me know if this sounds reasonable!

**Comment by petrelharp** [comment]
Ok - so it should be clear somewhere that generation time is time between mitotic divisions. And, since mutation rate and *recombination rate* are in units of "per generation", are those correct here?

**Comment by izabelcavassim** [comment]
also, don't we assume generation time to be in years?

**Comment by aays** [comment]
> Ok - so it should be clear somewhere that generation time is time between mitotic divisions. And, since mutation rate and _recombination rate_ are in units of "per generation", are those correct here?

They are! Mutation rate was calculated directly from MA experiments, but the recombination rates listed here are _population_ recombination rates (e.g. Ne * r calculated from genetic diversity) instead of estimates from lab crosses. I elaborated how I dealt with facultative sex a bit further in #863, reproduced here:

> An important note re: the recombination rate - sex takes place once every ~840 generations in the species (it's facultatively sexual), and the recombination rate estimates in here reflect that (e.g. they are r = rho / 2Ne, where rho was estimated directly from natural populations). To get 'true' physical recombination rate estimates, each of the values would be divided by the rate of sex, 0.001194 (explained further in the results/discussion [here](https://academic.oup.com/gbe/article/12/4/370/5809146?login=true#supplementary-data)) which I've left as a note in the script. I figure that if simulating wild populations it makes more sense to have effective recombination rates in there as opposed to the physical rates, since that would otherwise inflate the amount of recombination happening in the simulation.

So these are still technically per-generation per-bp recombination rates (e.g. the `mean_r_realized` column in the figshare dataset) where the infrequency of sex is 'baked into' the values via Ne and so they are accordingly lower. I figured this was the easiest way to do handle this based on the discussion in #857. 

Re: generation time, given the above calculation of 876 doublings in a year I set the generation time to 1/876 years, following how the generation time was set for E. coli (26280 doublings per year yielding a 1/26280 generation time).

**Comment by izabelcavassim** [comment]
ok @petrelharp made the edits you suggested and fixed a tiny thing on the generation time reference. I guess you can merge when you will.

**Comment by petrelharp** [comment]
Awesome, thanks!

**Pull request merged**